python/common/org/python/types/Int.java

```diff
@@ -108,9 +108,15 @@ public Int(long value) {
         } else if (other instanceof org.python.types.Float) {
             return new org.python.types.Bool(((double) this.value) == ((org.python.types.Float) other).value);
         } else if (other instanceof org.python.types.Bool) {
-            return new org.python.types.Bool(
-                (this.value == 0 && !((org.python.types.Bool) other).value)
-            );
+            if(this.value==0)
```

Comment:
Two minor style issues - there should be a space after the if, and if statements should *always* have braces, even if syntactically they aren't strictly required.